styles.css

```diff
@@ -10,23 +10,53 @@
   box-sizing: border-box;
 }
 
+.nav-bar.active {
+  background-color: white;
+  position: fixed;
+  top: 0;
+}
+
+.mobile-navhead {
+  padding: 1rem;
+}
+
+.mobile-navhead.active {
+  padding: 22px;
+}
+
```

Comment:
- [ ] Please add a close(X) button icon instead of a hamburger icon when the mobile menu is viewed to match the given design.
- [ ] Kindly add the correct margin value between the mobile menu options by increasing it slightly to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/203302025-1628574a-074b-45c4-b05d-a656292113f4.png) | ![image](https://user-images.githubusercontent.com/68796232/203301992-b4b8151a-3ac3-4e65-9573-ec37be923645.png) |